packages/web-react/src/components/Button/Button.tsx

```diff
@@ -2,42 +2,40 @@ import React, { ElementType, ForwardedRef, forwardRef } from 'react';
 import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
 import { SpiritButtonProps } from '../../types';
-import { useButtonAriaProps } from './useButtonAriaProps';
 import { useButtonStyleProps } from './useButtonStyleProps';
+import { useButtonAriaProps } from './useButtonAriaProps';
 
 const defaultProps = {
   color: 'primary',
-  type: 'button',
   isBlock: false,
   isDisabled: false,
   isSquare: false,
-  elementType: 'button',
   size: 'medium',
+  type: 'button',
 };
 
-export const Button = forwardRef<HTMLButtonElement, SpiritButtonProps>(
-  <T extends ElementType = 'button', C = void, S = void>(
-    props: SpiritButtonProps<T, C, S>,
-    ref: ForwardedRef<HTMLButtonElement>,
-  ): JSX.Element => {
-    const { elementType: ElementTag = 'button', children, ...restProps } = props;
-    const { buttonProps } = useButtonAriaProps(props);
-    const { classProps, props: modifiedProps } = useButtonStyleProps(restProps);
-    const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+const _Button = <T extends ElementType = 'button', C = void, S = void>(
+  props: SpiritButtonProps<T, C, S>,
+  ref: ForwardedRef<HTMLButtonElement>,
+) => {
+  const { elementType: ElementTag = 'button', ...restProps } = props;
+
+  const { buttonProps } = useButtonAriaProps(restProps);
+  const { classProps, props: modifiedProps } = useButtonStyleProps(restProps);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <ElementTag
+      {...otherProps}
+      {...buttonProps}
+      ref={ref}
+      className={classNames(classProps, styleProps.className)}
```

Comment:
no styleprops again because the object contains other items too?

Reply:
`style={styleProps.style}` is right below that

Reply:
If those styles weren't there, I wouldn't pass the tests